### ISSUE TITLE
Include the option "next reboot"

### ISSUE THREAD

**Issue by NeighNeighNeigh** [dieskim/FilevaultReboot#3]
Hi thanks for the nice tool.

Would it be possible to include the option "next reboot"
Where the app will not actually initiate a reboot or shutdown immediately.

I currently do this manually in the terminal with the following command:
`fdesetup authrestart -delayminutes -1` 

This means I can restart my machine later and it will do so with FileVault Authenticated

This would be a nice option here :)

**Comment by unstatusthequo** [comment]
Yes, I'd love this to work with OS updates. Right now it's a race condition to start update and enter credentials here.
